packages/react/src/components/table/table-row.tsx

```diff
@@ -57,7 +57,7 @@ function getCellBackgroundCss({
 
     return css`
         &:hover td {
-            background-color: ${theme.greys.grey};
+            background-color: ${theme.component['table-row-cell-hover-background-color']};
```

Comment:
Un cell est toujours dans une row, c'est peut-être superflu de mettre `-row-` dans le token pour la cell
```suggestion
            background-color: ${theme.component['table-cell-hover-background-color']};
```